create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/conversion/PublicationContextCreator.java

```diff
@@ -107,6 +131,14 @@ public Book createReport() {
                 -> new Report(bookSeries, seriesTitle, seriesNumber, publishingHouse, isbnList)).orElseThrow();
     }
 
+    public Chapter createChapter() {
+        //Todo: hvordan få tak i en URI min chapter er partOf?
+//        Chapter.Builder chapterBuilder = new Chapter.Builder();
+//        URI partOfUri = null;
+//        return attempt(() -> chapterBuilder.withPartOf(partOfUri).build()).orElseThrow();
+        return attempt(Chapter::new).orElseThrow();
```

Comment:
Hvis publicationContext er påkrevd for å importere en xml fil, så tror jeg ikke vi kan sluke exceptionet her. Det tror jeg bare bør kastes videre.